### ISSUE TITLE
Broken pseudo selector body if it contains more than one style attribute

### ISSUE THREAD

**Issue by ranzwertig** [davidspinat/stylebuddy#21]
Using pseudo selectors we found an issue. Whenever we have more than one style attribute inside the pseudo selector its not rendered properly.

e.g. 

~~~
foo: {
  ':hover': {
    color: '#ffffff',
    display: 'block',
    height: 0
  }
}
~~~

leads to:

~~~
.foo:hover {
    color: #ffffff;
    , display: block;
    , height: 0;
}
~~~

I also provided #20 which fixes this.

**Comment by davidspinat** [comment]
Thanks for reporting this issue and providing your pull request!

I just published version `1.1.1` which includes your fix.

Cheers

**Comment by ranzwertig** [comment]
@davidspinat thanks a lot for beeing so responsive :)
